Release 1.5.0 (#293)
audeering/audb: CHANGELOG.rst

and this project adheres to `Semantic Versioning`_.

Version 1.5.0 (2023-04-27)
--------------------------

* Added: support loading and publishing
  of database attachments
  (``audformat.Attachment``)
* Added: ``audb.load_attachment()``

>   to load only an attachment of a database

Reviewer: ```suggestion
  to load a single attachment of a database
```

* Added: ``audb.info.attachments()``
  to return the attachments entry
  of a database header
* Added: ``attachments`` argument to ``audb.load()``

>   to filter attachments
>   to be loaded with the database

Reviewer: ```suggestion
  to load only specific
  attachments of a database
```

* Changed: raise ``RuntimeError`` in ``audb.publish()``

>   if any media file
>   contains uppercase letters
>   in its file extension

Reviewer: ```suggestion
  if the file extension of a media file
  contains uppercase letters
```
